### PR TITLE
 Add wired support to Huawei 

### PR DESCRIPTION
**Pull request by julsemaan**
For review and merge

Thanks

### CODE REVIEW

File: lib/pf/Switch/Huawei/S5710.pm [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+=item handleReAssignVlanTrapForWiredMacAuth
+
+Called when a ReAssignVlan trap is received for a switch-port in Wired MAC Authentication.
+
+=cut
+
+sub handleReAssignVlanTrapForWiredMacAuth {
+    my ($this, $ifIndex, $mac) = @_;
+    my $logger = Log::Log4perl::get_logger(ref($this));
```
Comment (dwlfrth) [inline lib/pf/Switch/Huawei/S5710.pm]: Shouldn't we use __PACKAGE__ rather than ref($this) ?

File: lib/pf/Switch/Huawei/S5710.pm [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+sub description { 'Huawei S5710' }
+
+=head1 SUBROUTINES
+
+=over
+
+=cut
+sub supportsWiredMacAuth { return $TRUE; }
+sub supportsWiredDot1x { return $TRUE; }
+=item getIfType
```
Comment (dwlfrth) [inline lib/pf/Switch/Huawei/S5710.pm:40]: Missing a space before that item block :P

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 3]
```diff
@@ -1913,6 +1913,73 @@ RADIUS configuration
...
+
+l2protocol-tunnel user-defined-protocol 802.1x protocol-mac 0180-c200-0003 group-mac 0100-0000-0002
+domain pf
+
+dot1x enable
+dot1x dhcp-trigger
+
+radius-server template packetfence
+ radius-server shared-key cipher <yourSecret>
+ radius-server authentication 172.20.20.109 1812
```
Comment (fdurand) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: in this doc pf is at 192.168.1.5

**Comment by julsemaan** [comment]
Had to merge branch manually
Commit ce5a0dc14759d5316f9fa032e01665bd5b2f75af
Closing PR

**Pull request closed without merging**